bdleon/nashville-sell-and-trade-client
# Bd product list

#### Changes Made

1. Modified file `ProductManager` to include `fetch` and functionality.
2. Modified file `ProductList` to include `react-bootstrap` and functionality.
3. Modified file `ProductList` to include `updated handler` and functionality
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout bd-product-list
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > click through filter and maker sure they re-render when checking through options
    > Example: When user does BLANK, then BLANK should happen.
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
